opentelemetry-jaeger/src/thrift/agent.rs

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
Hm this file should have rustfmt skipped via `#![cfg_attr(rustfmt, rustfmt_skip)]`, can probably remove this change as this is all generated code from thrift. Harder to diff if the thrift compiler output changes from version to version if we change their formatting.

Reply:
Sure. Reverted. I think if you want to get `rustfmt` to skip a file you can add `#[rustfmt::skip]` in `lib.rs` where the module is declared. I found the answer on [stackoverflow](https://stackoverflow.com/questions/59247458/is-there-a-stable-way-to-tell-rustfmt-to-skip-an-entire-file#:~:text=1%20Answer&text=Currently%20Rustfmt%20traverses%20the%20mod,dont_format_this_file.rs%20will%20be%20skipped.) and tried it out. It seems to work. Although it does say it may not work with rustfmt 2...